code/src/components/Loading/Loading.css

```diff
@@ -0,0 +1,42 @@
+.loading-overlay {
```

Comment:
You choose to not make an Js file with all the details for the spinner, why not?